addons/mail/static/src/owl/components/activity/activity.xml

```diff
@@ -0,0 +1,147 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="mail.component.Activity" owl="1">
+        <t t-if="props.activityLocalId and storeProps.activity">
+            <div class="o_Activity">
+                <div class="o_Activity_sidebar">
+                    <div class="o_Activity_user">
+                        <img class="o_Activity_userAvatar rounded-circle"
+                             t-att-class="{
+                                o_Message_authorRedirect: hasAuthorRedirect,
+                                o_redirect: hasAuthorRedirect
+                             }"
+                             t-att-src="assignedAvatarSource"
+                             t-att-alt="storeProps.activity.userDisplayName"/>
+                        <i class="o_Activity_userAvatar_icon fa" t-att-class="iconClasses"/>
+                    </div>
+                </div>
+                <div class="o_Activity_core">
+                    <div class="o_Activity_info">
+                        <span class="o_Activity_delay"
+                              t-att-class="'o_Activity_color_'+storeProps.activity.activityState">
+                            <t t-esc="delayLabel"/>
+                        </span>
+                        :
+                        <t t-if="storeProps.activity.summary">
+                            <span class="o_Activity_summary">
+                                &#8220;<t t-esc="storeProps.activity.summary"/>&#8221;
```

Comment:
Prefer using a getter that includes the quotes.
2 advantages:
- no need to master HTML code of quotes to read template
- quotes are locale-dependent (guillemets « » in french, かぎかっこ 「 」in japanese ), so this may be a good idea to translate with `this.env._t("%s")`